napari/components/overlays/brush_circle.py

```diff
@@ -13,7 +13,10 @@ class BrushCircleOverlay(CanvasOverlay):
         The diameter of the brush circle in canvas pixels.
     position : Tuple[int, int]
         The position (x, y) of the center of the brush circle on the canvas.
+    frozen_pos : bool
+        If True, the overlay does not respond to mouse movements.
     """
 
     size: int = 10
     position: Tuple[int, int] = (0, 0)
+    frozen_pos: bool = False
```

Comment:
```suggestion
    position_is_frozen: bool = False
```

nit for readability, non blocking

Reply:
It looks clearer, I changed it. Initially, I tried to make the variable length shorter as `self.viewer._brush_circle_overlay.position_is_frozen` is too long. With the new name, I had to refactor a little bit the code in Canvas._on_cursor to avoid to it.